tests/puppeteer/pages/BO/product.js

```diff
@@ -1,13 +1,126 @@
+// Using chai
```

Comment:
The name of the file is `product.js` and it's for the `Products` listing page which means we might have a conflict in naming when we'll want to create a file for a single product... what do you think ?

Reply:
done thanks